client/lib/promote-post/index.ts

```diff
@@ -35,11 +35,12 @@ export async function loadDSPWidgetJS( onLoad?: () => void ) {
 	document.body.appendChild( script );
 }
 
-export async function showDSPWidgetModal( siteId: number, postId?: number ) {
+export async function showDSPWidgetModal( siteId: number | string, postId?: number ) {
 	if ( ! window.BlazePress ) {
 		await loadDSPWidgetJS( async () => await showDSPWidgetModal( siteId, postId ) );
 	} else {
 		await window.BlazePress.render( {
+			domNodeId: 'promote__widget-container',
```

Comment:
This is hardcoded, probably should be added as a parameter to `showDSPWidgetModal`